### PR TITLE
mwan3: fix package to dependency

### PR DESCRIPTION
**Pull request by feckert**
Maintainer: many

Description:
Update jshn package dependencies

### CODE REVIEW

**Comment by dibdot** [comment]
what will be fixed actually? I think jshn is pretty much standard, I've never received any complaints about missing dependencies ... e.g. core components like netifd depends on jshn, too.

**Comment by feckert** [comment]
@dibdot You are right it is pretty much standard but not a "standard". If owrt/lede decide to change the json parser (which i think will not happen in the near future but who knows) then the package is not installed by default anymore.
Where do we stop to think this is worth to add or not to add to the package dependency list?
I think if a package has a dependency then we should add it to the Makefile.

**Comment by dibdot** [comment]
@feckert If we go this direction, than 'jshn' is not enough. There are more core dependencies like 'jsonfilter', 'busybox' and of course 'base-files' - at least for travelmate and adblock. I am not convinced, but maybe your commit is a good starting point.

**Comment by tpham3783** [comment]
Hmm,

I thought jsn.sh is part of libubox.  I did all of this work (ref.
https://github.com/tpham3783/libubox ) just to be able to use
bash-json-parser on other linux distros..

**Comment by feckert** [comment]
@tpham3783 sorry i do not understand your statement? Do you think we need the dependency?

@dibdot 

- base-files is a common dependency this files are also exist in debian based distros and they are not mentioned in every package.
- jsonfilter is not a common package and this should be mentioned.
- busybox is complicated because this package is a multi binary and the dependency must be fixed on compile time, If you compile your own distribution its OK but if you use a precompiled distribution from "lede" this will definitely fails if you add all busybox dependency.

Maybe we should add a dummy package which reference the dependencies to the corepackages?

**Comment by dibdot** [comment]
@feckert from my point of view it's sufficient to add jsonfilter to your orginal PR (for adblock & travelmate) ... the other dependencies are even more standard for LEDE images.
Thanks!

**Comment by feckert** [comment]
I have added in net/mwan3 the dependency to the "ip-full" package. Which was a solution for the following issue.
#5022

**Comment by schuza** [comment]
What is missing to get the mwan3 commit 2598f8e merged into trunk? At least the missing dependency "ip-full" should to be added in the meantime.

**Comment by feckert** [comment]
@schuza i think i have to update adblock and ntpd again
I will have to this tomorrow

**Comment by feckert** [comment]
@schuza update PR no conflicts anymore

**Comment by feckert** [comment]
@hnyman can we merge this?

**Comment by camelcamro** [comment]
@admin's: pls can you merge & pull into the builds ?
Devel did everything fine from his side

File: net/mwan3/Makefile [hunk 1]
```diff
@@ -19,7 +19,13 @@ define Package/mwan3
    SECTION:=net
    CATEGORY:=Network
    SUBMENU:=Routing and Redirection
-   DEPENDS:=+ip +ipset +iptables +iptables-mod-conntrack-extra +iptables-mod-ipopt
+   DEPENDS:= \
+     +ip-full \
```
Comment (karlp) [inline net/mwan3/Makefile]: You don't mention in your commit note _this_ change in dependencies!

**Pull request merged**